installation/scripts/build-kyma-installer.sh

```diff
@@ -5,6 +5,13 @@ set -o errexit
 CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 ROOT_DIR=${CURRENT_DIR}/../../
 IMAGE_NAME="$(${CURRENT_DIR}/extract-kyma-installer-image.sh)"
+INSTALLER_VERSION='9da43968'
```

Comment:
i would  prefer that INSTALLER_VERSION will stay in dockerfile. this way anyone can build kyma-installer not using bash script.

Reply:
done!